dali/operators/ssd/box_encoder.cuh

```diff
@@ -68,6 +68,7 @@ class BoxEncoder<GPUBackend> : public Operator<GPUBackend> {
     stds_.Resize({BoundingBox::size}, DALI_FLOAT);
     auto stds_data = stds_.mutable_data<float>();
     MemCopy(stds_data, stds.data(), BoundingBox::size * sizeof(float));
+    cudaStreamSynchronize(0);
```

Comment:
Little nitpick, `0` integer constant used as `nullptr`

Reply:
In this case 0 is preferable - it means "default stream", not "null stream" - this has already been a source of confusion and led to some bad designs.